### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by rezabekf**
Add issue template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (stilvoid) [inline .github/ISSUE_TEMPLATE/feature_request.md:11]:
```suggestion
A clear and concise description of what the problem is [e.g. I'm always frustrated when...]
```

Comment (stilvoid) [reply .github/ISSUE_TEMPLATE/feature_request.md:11]: Just for consistency :)

**Review by stilvoid — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (rezabekf) [reply .github/ISSUE_TEMPLATE/feature_request.md:11]: Nice one cheers, fixed!

**Review by stilvoid — APPROVED** [review]
lgtm :+1:

**Pull request merged**